.ci/deps.sh

```diff
@@ -16,4 +15,4 @@ for dep_version in "${dep_versions[@]}" ; do
 
   pip3 install -q -r test-requirements.txt
   pip3 install -q -r requirements.txt
-done
\ No newline at end of file
+done
```

Comment:
@Abhisek- Or you could add another commit fixing just that (although I'm leaning towards a end-of-line bear too). The idea is that each commit should be about one thing and just that one thing. Makes it much easier later on when you go through your commits.